service/sg_accel_service_uninstall.sh

```diff
@@ -86,7 +88,7 @@ case $OS in
                 	rm /etc/init/${SERVICE_NAME}.conf
                 fi
                 ;;
-            16)
+            16|17)
```

Comment:
This seems difficult to maintain .. I wonder if it makes sense to just say "If >= 16" here?